src/frontend/pages/game.js

```diff
@@ -5,6 +5,10 @@ import React, { useState } from 'react';
 
 export default function Home() {
   const [takeVal, setTakeVal] = useState(10);
+  // state variable to track which color this user is once we hook this up to back end this will be dynamic
+  const [thisColor, setThisColor] = useState('Green');
```

Comment:
Also, can we add green as a constant something like `const defaultPlayerColor = 'Green'`